app/controllers/holidays_controller.rb

```diff
@@ -0,0 +1,18 @@
+class HolidaysController < ApplicationController
+  before_action :authorise_for_managing_resources!
+
+  def index
+    respond_to do |format|
+      format.html
+      format.json do
+        render json: Holiday.merged_for_calendar_view
```

Comment:
This won't support the `start` and `end` dates provided by the calendar when the bindings (day, week, month, forward and backward) change, right?